packages/effect/src/Effect.ts

```diff
@@ -4749,7 +4749,7 @@ export const runFork: <E, A>(effect: Effect<never, E, A>) => Fiber.RuntimeFiber<
  */
 export const runCallback: <E, A>(
   effect: Effect<never, E, A>,
-  onExit?: (exit: Exit.Exit<E, A>) => void
+  options?: Runtime.RunCallbackOptions<E, A> | undefined
```

Comment:
Will need to add a minor changeset for this.

Reply:
Updated